tests/Database/SelectFields/AlwaysRelationTests/AlwaysRelationTest.php

```diff
@@ -106,8 +181,8 @@ protected function getEnvironmentSetUp($app)
         $app['config']->set('graphql.schemas.custom', null);
 
         $app['config']->set('graphql.types', [
-            LikableInterfaceType::class,
             CommentType::class,
+            LikableInterfaceType::class,
```

Comment:
Uh, when I changed the order it broke the tests because `LikableInterface` couldn't be found (will revert)

@ZaidBarghouthi where you aware of this? feels weird that this is order dependent. I understand why, in the code, but never saw this before.

Reply:
No, I had no idea!